samples/wifi/shell/README.rst

```diff
@@ -71,24 +71,25 @@ Supported CLI commands
    * - Subcommands
      - Description
    * - scan
-     - Scan for access points in the vicinity
+     - Scan for Wi-Fi APs
    * - connect
-     - | Connect to an access point with below parameters:
+     - | Connect to Wi-Fi AP with below parameters:
        | <SSID>
        | <Channel number> (optional: 0 means all)
        | <PSK> (optional: valid only for secured SSIDs)
        | <Security type> (optional: 0-None, 1-PSK, 2-PSK-256, 3-SAE)
        | <MFP> (optional: 0-Disable, 1-Optional, 2-Required)
    * - disconnect
-     - Disconnect from the current access point
+     - Disconnect from the Wi-Fi AP
    * - status
-     - Get the status of the Wi-Fi interface
+     - Status of the Wi-Fi interface
    * - statistics
-     - Get the statistics of the Wi-Fi interface
-   * - ap_enable
-     - Configure the Wi-Fi interface as access point mode
-   * - ap_disable
-     - Configure the Wi-Fi interface as station mode
+     - Wi-Fi interface statistics
+   * - ap
+     - | Access Point mode commands
+       | enable - Configure the Wi-Fi interface as access point mode
```

Comment:
Enable Access Point mode, with the following parameters: 
<List parameters from the shell>